<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Markov processes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
     <property name="title">
      <string>Input data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStatesCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
        <font>
        <family>Courier New</family>
        <pointsize>14</pointsize>
        </font>
        </property>
        <property name="text">
         <string>Number of states:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxStatesCount"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTableWidget" name="tableWidgetMatrix"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
     <property name="title">
      <string>Result</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonSolve">
       <property name="font">
        <font>
        <family>Courier New</family>
        <pointsize>14</pointsize>
        </font>
        </property>
        <property name="text">
         <string>Ð¡alculate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetSolution">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
